.travis.yml

```diff
@@ -6,7 +6,7 @@ script:
   - xcrun instruments -t 'Blank' -l 10 -w "iPhone 6s (10.0)"
   - swift build
   - swift test
-  - xcodebuild -scheme JSONUtilities -configuration Debug -sdk iphonesimulator10.0 -destination "OS=10.0,name=iPhone 6s" test -enableCodeCoverage YES
+  - xcodebuild -scheme "JSONUtilities iOS" -configuration Debug -sdk iphonesimulator10.0 -destination "OS=10.0,name=iPhone 6s" test -enableCodeCoverage YES
```

Comment:
I think that since we now have 4 schemes(and targets) we should probably build all 4 of them to make sure that they don't break